experiments/TestCase/baroclinic_wave.jl

```diff
@@ -296,15 +297,36 @@ function config_diagnostics(FT, driver_config)
     _planet_radius = FT(planet_radius(param_set))
 
     info = driver_config.config_info
+
+    # Setup diagnostic grid(s)
+    nlats = 128
+
+    sinθ, wts = compute_gaussian!(nlats)
+    lats = asin.(sinθ) .* 180 / π
+    lons = 180.0 ./ nlats * collect(FT, 1:1:(2nlats))[:] .- 180.0
+
     boundaries = [
-        FT(-90.0) FT(-180.0) _planet_radius
-        FT(90.0) FT(180.0) FT(_planet_radius + info.domain_height)
+        FT(lats[1]) FT(lons[1]) _planet_radius
+        FT(lats[end]) FT(lons[end]) FT(_planet_radius + info.domain_height)
     ]
-    resolution = (FT(2), FT(2), FT(1000)) # in (deg, deg, m)
+
+    lvls = collect(range(
+        boundaries[1, 3],
+        boundaries[2, 3],
+        step = FT(1000), # in m
+    ))
+
+    #boundaries = [
+    #    FT(-90.0) FT(-180.0) _planet_radius
+    #    FT(90.0) FT(180.0) FT(_planet_radius + info.domain_height)
+    #]
+
+    #resolution = (FT(2), FT(2), FT(1000)) # in (deg, deg, m)
+
```

Comment:
L319-324: Could these lines be deleted or do we want to keep it commented out as a record?

Reply:
I kept it in in case someone didn't want to use the Gaussian grid :)

Reply:
> I kept it in in case someone didn't want to use the Gaussian grid :)

haha that's how I figured~ sounds good

I'm running it now and will get the outputs/plots very soon :P